docs/data/base/getting-started/customization/StylingSlotsSingleComponent.tsx

```diff
@@ -1,8 +1,8 @@
 import * as React from 'react';
 import { styled } from '@mui/system';
-import { Switch, switchClasses } from '@mui/base/Switch';
+import { Switch as UnstyledSwitch, switchClasses } from '@mui/base/Switch';
 
-const StyledSwitch = styled(Switch)`
+const Switch = styled(UnstyledSwitch)`
```

Comment:
this makes a lot more sense to me. These components are a lot more likely to be used called `Switch` in the final app pages.

Reply:
I usually use `BaseSwitch` in these contexts. Would it work for you as well?

Reply:
I think we can use this instance to come with a standard for all the demos in the documentation. I see two decisions we can make:

1. prefix vs. suffit: BaseSwitch vs. SwitchBase
2. Name: UnstyledSwich vs. BaseSwitch vs. else

Prefix makes sense for me, it makes it easier to spot what's styled and what's not.
Base references the projet name, it makes sense.

I think we can bring in @samuelsycamore, @mnajdova, @mj12albert on this discussion, they have also created demos.

Reply:
I like `BaseSwitch` the best 👍

Would it be `const Switch = styled(BaseSwitch)` then? That looks great to me.

Reply:
Alright `BaseSwitch`, I'm renaming it in this PR. I think that we should update the rest of the docs / codebase. I have created https://github.com/mui/material-ui/issues/38904 to keep track of it.